packages/strapi-plugin-versioning/README.md

```diff
@@ -0,0 +1,84 @@
+# Strapi - Versioning Plugin (bookshelf only)
+
+Utilities for [Strapi Headless CMS](https://github.com/strapi/strapi) that allow
+searching across nested structures including components.
+
+### ⏳ Installation
+
+(Use **yarn** to install this plugin within your Strapi project (recommended).
+[Install yarn with these docs](https://yarnpkg.com/lang/en/docs/install/).)
+
+```bash
+yarn add strapi-plugin-versioning
```

Comment:
if You would like to publish it, please specify exact version of working strapi and describe known problems with other versions

Reply:
added